test/networkd-test.py

```diff
@@ -39,18 +40,70 @@
 import shutil
 import socket
 
-networkd_active = subprocess.call(['systemctl', 'is-active', '--quiet',
-                                   'systemd-networkd']) == 0
-have_dnsmasq = shutil.which('dnsmasq')
+HAVE_DNSMASQ = shutil.which('dnsmasq') is not None
+
+NETWORK_UNITDIR = '/run/systemd/network'
+
+NETWORKD_WAIT_ONLINE = shutil.which('systemd-networkd-wait-online',
+                                    path='/usr/lib/systemd:/lib/systemd')
```

Comment:
Running pylint on the file considers those variables to be constants, so it complains when they are not in caps-with-underscore naming format.  https://www.python.org/dev/peps/pep-0008/#constants

I'll make the other changes you suggest but leave this as is (unless you confirm these should go against pylint's detection before I push the amended commit).